.travis.yml

```diff
@@ -4,7 +4,6 @@ rvm:
   - 2.4.0
   - 2.3.3
   - 2.2.6
-  - 2.1.10
```

Comment:
RIP!
FYI, 2.2 is also not maintained https://www.ruby-lang.org/en/news/2018/06/20/support-of-ruby-2-2-has-ended/ and 2.3 is no longer built in https://build.opensuse.org/project/show/devel:languages:ruby:extensions

Reply:
I am keeping it to still have option to backport newer CFA to old released products if needed. So as long as it works, I keep it there just to see.